doc/chips/10.rst

```diff
@@ -528,4 +533,4 @@ to the initializer.  The type designer may also utilize ``noinit`` on individual
 fields during the first phase of the initializer body.  If there are operations
 which must occur on a copy made by the compiler before it is operated on by
 other code, such as via an ``in`` intent, the type designer may supply a copy
```

Comment:
'such as via an in intent` seems to modify 'before it is operated on by other code'. You might just leave out the phrase.